engine/src/test/scala/cromwell/webservice/MetadataBuilderActorSpec.scala

```diff
@@ -255,11 +249,11 @@ class MetadataBuilderActorSpec extends TestKit(ActorSystem("Metadata")) with Fla
 
   it should "return workflow status" in {
     val workflow = WorkflowId.randomId()
-    val query = MetadataQuery(Option(workflow), None, Option("status"))
+    val query = MetadataQuery(workflow, None, Option("status"))
     val events = List(
-      MetadataEvent(MetadataKey(workflow, None, "status"), MetadataValue("Running"), DateTime.parse("2000-01-02")),
-      MetadataEvent(MetadataKey(workflow, None, "status"), MetadataValue("Done"), DateTime.parse("2000-01-03")),
-      MetadataEvent(MetadataKey(workflow, None, "status"), MetadataValue("Submitted"), DateTime.parse("2000-01-01"))
+      MetadataEvent(MetadataKey(workflow, None, "status"), MetadataValue("Running"), Timestamp.valueOf("2000-01-02 12:00:00")),
+      MetadataEvent(MetadataKey(workflow, None, "status"), MetadataValue("Done"), Timestamp.valueOf("2000-01-03 12:00:00")),
+      MetadataEvent(MetadataKey(workflow, None, "status"), MetadataValue("Submitted"), Timestamp.valueOf("2000-01-01 12:00:00"))
```

Comment:
By setting these timestamps to be the same, are you deliberately making this non-deterministic?